<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Noise</class>
 <widget class="QWidget" name="Noise">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Noise</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="primaryDenoiser_gb">
     <property name="title">
      <string>Primary Denoiser</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="primaryDenoiser_Strength_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Greyscale threshold</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Strength</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="primaryDenoiser_GrayscaleThreshold_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="sharpening_gb">
     <property name="title">
      <string>Sharpening</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Radius</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Threshold</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Strength</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="sharpening_Radius_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="sharpening_Threshold_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="sharpening_Strength_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Detail</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="sharpening_Detail_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="noiseSuppression_gb">
     <property name="title">
      <string>Noise Suppression</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Edge Avoidance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Strength</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="noiseSuppression_EdgeAvoidance_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="noiseSuppression_Strength_sb">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="DNSenable_cb">
     <property name="text">
      <string>DNS enable (instant)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>DNSenable_cb</tabstop>
  <tabstop>primaryDenoiser_Strength_sb</tabstop>
  <tabstop>primaryDenoiser_GrayscaleThreshold_sb</tabstop>
  <tabstop>sharpening_Radius_sb</tabstop>
  <tabstop>sharpening_Strength_sb</tabstop>
  <tabstop>sharpening_Threshold_sb</tabstop>
  <tabstop>sharpening_Detail_sb</tabstop>
  <tabstop>noiseSuppression_EdgeAvoidance_sb</tabstop>
  <tabstop>noiseSuppression_Strength_sb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
